backend/backend.js

```diff
@@ -7,7 +8,7 @@ const editorPrefix = 'editor';
 /* @expose */
 export async function getHtmlFields() {
   try {
-    const { html, css, js } = await get(editorPrefix);
+    const { html, css, js } = (await get(editorPrefix)) || initialData;
```

Comment:
It looks like the code is repeating between `backend/backend.js` and `backend/_handler.js` - can it be extracted into a single location (e.g. so we don't define `editorPrefix` twice).

Reply:
I exported the editorPrefix and importing in handler - hope this is what you meant :)